src/test/java/org/retest/rebazer/domain/PullRequestTest.java

```diff
@@ -0,0 +1,26 @@
+package org.retest.rebazer.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class PullRequestTest {
+
+	private PullRequest pullRequest;
+
+	@Before
+	public void setUp() throws Exception {
+		pullRequest = new PullRequest( 1, "source", "destination", "lastUpdate" );
+	}
+
+	@Test
+	public void toString_should_return_correct_string() {
+		assertThat( pullRequest.toString() ).isEqualTo( "PR #1 (source -> destination)" );
```

Comment:
There is `shouldHaveToString`: http://joel-costigliola.github.io/assertj/core-8/api/org/assertj/core/error/ShouldHaveToString.html